test/test_distributions.py

```diff
@@ -389,6 +389,12 @@ def is_all_nan(tensor):
             'high': torch.tensor([2.0, 3.0], requires_grad=True),
         },
     ]),
+    Example(Weibull, [
+        {
+            'scale': torch.tensor(torch.randn(5, 5).abs(), requires_grad=True),
+            'shape': torch.tensor(torch.randn(1).abs(), requires_grad=True)
```

Comment:
Can we name this parameter `concentration` rather than `shape`? Two reasons:
1. tensorflow distributions calls this `concentration`, and we want to maintain compatibility
2. `shape` usually refers to a `torch.Size` of some tensor, and it would be nice to avoid confusing users.